projects/plugins/jetpack/modules/google-fonts/current/class-jetpack-google-font-face.php

```diff
@@ -40,9 +40,13 @@ public function wp_loaded() {
 	}
 
 	/**
-	 * Turn off hooks to print fonts on wp-admin
+	 * Turn off hooks to print fonts on wp-admin, except for GB editor pages.
 	 */
-	public function admin_init() {
+	public function current_screen() {
+		if ( $this->is_block_editor() ) {
+			return;
+		}
+
 		remove_action( 'admin_print_styles', 'wp_print_fonts', 50 );
```

Comment:
We have to turn off this to avoid printing the font CSS twice.

Reply:
I've updated it here https://github.com/Automattic/jetpack/pull/34776/commits/5f45928d03d435d979a8ac01fb36611bce5042f5